examples/src/Example/APIGateway.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE PartialTypeSignatures #-}
+
+{-# OPTIONS_GHC -fno-warn-partial-type-signatures #-}
+{-# OPTIONS_GHC -fno-warn-name-shadowing #-}
+{-# OPTIONS_GHC -fno-warn-unused-imports #-}
+
+module Example.APIGateway where
+
+import           Control.Lens
+import           Control.Monad.IO.Class
+import           Control.Monad.Trans.AWS
+import           Data.Monoid
+import           Network.AWS.Data
+import           Network.AWS.APIGateway
+import           System.IO
+
+main :: IO ()
+main = do
+  lgr <- newLogger Trace stdout
```

Comment:
Hate to bikeshed :trollface:  but maybe 4 spaces here, if only because the rest of the project is the same.